mlsdbtool/main.cpp

```diff
@@ -1536,4 +1646,4 @@ int main(int argc, char *argv[])
 
     printUsageError();
     return 1;
-}
+}
```

Comment:
nitpick: please add a newline character at the end of file (e.g. opening with vi(m) and saving the file is sufficient)

Reply:
New line is added.